components/gc-subway/docs-en.html

```diff
@@ -27,14 +27,24 @@ <h2 id="purpose">Purpose</h2>
 <p>Break up long and complex content into pages that each focus on a step or specific answer people need before moving to the next step or section. Ideal for presenting services and processes.</p>
 
 <h2 id="workingExamples">Working examples</h2>
-<ul>
-	<li><a href="index-en.html">Index page</a></li>
-	<li><a href="section1-en.html">Section page</a></li>
-	<li><a href="sub-section-en.html">Sub-section page (default)</a></li>
-	<li><a href="sub-section-noline-en.html">Sub-section page (no line)</a></li>
-	<li><a href="section-subtitle-en.html">Section page with custom subtitle</a></li>
-	<li><a href="section-end-en.html">Section page with section end element</a></li>
-	<li><a href="section-rel-info-en.html">Section page with related information</a></li>
+<div data-wb-jsonmanager='{
+	"url": "/components/gc-subway/index.json-ld",
```

Comment:
Use a relative path because this absolute path wont work on ghpage

```suggestion
	"url": "index.json-ld",
```